dist/info/fuse_libretro.info

```diff
@@ -20,28 +20,28 @@ database = "Sinclair - ZX Spectrum +3|Sinclair - ZX Spectrum"
 # BIOS / Firmware
 firmware_count = 8
 firmware0_desc = "128p-0.rom (Pentagon)"
-firmware0_path = "128p-0.rom"
+firmware0_path = "fuse/128p-0.rom"
 firmware0_opt = "true"
 firmware1_desc = "128p-1.rom (Pentagon machines)"
-firmware1_path = "128p-1.rom"
+firmware1_path = "fuse/128p-1.rom"
 firmware1_opt = "true"
 firmware2_desc = "trdos.rom (Pentagon machines)"
-firmware2_path = "trdos.rom"
+firmware2_path = "fuse/trdos.rom"
 firmware2_opt = "true"
 firmware3_desc = "gluck.rom (Pentagon 128K / 1024)"
```

Comment:
@tomconte Just a minor naming quibble 😄 . Should be -
`firmware3_desc = "gluck.rom (Pentagon 512K / 1024K)"`
as gluck.rom isn't used on Pentagon 128K.